### DISCUSSION TITLE
Guidance on Using DP-GEN for Pre-Generated Water-Pt Interface Structures

### DISCUSSION THREAD

**Discussion by poyaii** [deepmodeling/dpgen#1632] [Q&A]
Hello.
I would appreciate it if you could guide me. I'm working on the water-Pt interface. I performed ab-initio calculations without using DP-GEN. I have generated three structures: water, Pt, and the Pt-H2O interface, and I have carried out ab-initio calculations on them. I have recently become familiar with DP-GEN. Now, 1. considering that I didn't generate my structure using DP-GEN, should I use the 'simplify' section of DP-GEN? 2. Can 'simply' also provide good results in model accuracy like 'run'? 

Thank you.

**Comment by njzjz** [comment]
> Can 'simply' also provide good results in model accuracy like 'run'?

If `dpgen run` does not generate new data, it means `dpgen simplify` provides the same result as `run`; otherwise new data from `dpgen run` should be trained.
